app/views/contacts/edit.html.erb

```diff
@@ -1,11 +1,22 @@
-<div class="container mt-3" style="padding: 90px">
-  <div class="row justify-content-center">
-    <div class="col-12" %>
-      <h1>Edit <%= @contact.first_name %> <%= @contact.last_name %></h1>
+<div class="dashboard-container">
+  
+  <div class="sidebar col-sm-2">
+    <%= render 'shared/sidebar' %>
+  </div>
+
+  <div class="col-sm-10">
+    <div class="inner-navbar">
+      <%= render 'shared/navbar' %>
+    </div>
```

Comment:
If the sidebar and navbar are rendered on every page (when logged in), then perhaps they should be in the `application.html.erb` instead so they render on all pages by default. Otherwise you will have to manually add it in like this on every single view.

Reply:
Makes totaly sense. I will refactor it this weegk